### PR TITLE
aragonOS page

### PR DESCRIPTION
**Pull request by luisivan**
Requesting feedback for this

### CODE REVIEW

File: src/components/OS/Snippet/Snippet.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { styled, IllustratedSection } from '@aragon/ui'
+import OverlapBackground from '../../OverlapBackground/OverlapBackground'
+
+import code from './assets/code.png'
```
Comment (izqui) [inline src/components/OS/Snippet/Snippet.js]: I would use a more serious app such as the voting app for the code example :)

File: src/components/OS/Snippet/Snippet.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  img {
+    min-width: 512px;
+  }
+`
+
+const Snippet = () => (
+  <StyledOverlapBackground>
+    <IllustratedSection>
+      <Subtitle dark>
+        <p>Upgradeability, permissions and governance, all taken care of</p>
```
Comment (izqui) [inline src/components/OS/Snippet/Snippet.js]: permissions -> access control?

File: src/components/OS/Snippet/Snippet.js (hunk 1 above)
Comment (luisivan) [reply src/components/OS/Snippet/Snippet.js]: Tried to skim through it but I didn't find any snippet that would fit. Maybe we can have the contract is AragonApp thing, then cut everything else and display the roles, and show a function behind the ACL. What do you think?

File: src/components/OS/Snippet/Snippet.js (hunk 2 above)
Comment (luisivan) [reply src/components/OS/Snippet/Snippet.js]: Changed!

File: src/components/OS/Snippet/Snippet.js (hunk 1 above)
Comment (izqui) [reply src/components/OS/Snippet/Snippet.js]: hmmm i guess it is fine

Comment (luisivan) [reply src/components/OS/Snippet/Snippet.js]: LGTM

**Comment by luisivan** [comment]
@Smokyish awaiting for your approval

**Comment by Smokyish** [comment]
Will do this and #42 today!

File: src/components/OS/Docs/Docs.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+`
+
+const Docs = () => (
+  <Main>
+    <div>
+      <SectionTitle title="Getting started" />
+      <h1 className="title">
+        The aragonOS architecture
+      </h1>
+      <img src="https://cdn-images-1.medium.com/max/1200/1*5rZnVMu2_J8L2McuFNnwmA.gif" alt="" />
```
Comment (Smokyish) [inline src/components/OS/Docs/Docs.js]: Don't want to rely on Medium CDN, going to host the image in the wiki repo.

**Comment by Smokyish** [comment]
@luisivan @izqui I pushed my changes as a new commit, please (re-)review!

**Comment by luisivan** [comment]
@Smokyish agreed with most changes, pushed a new commit taking back a change that also modified meaning

**Comment by Smokyish** [comment]
@luisivan Cool, your commit added an extra "-" so it said "- They expose" and I also removed the period at the end for consistency.

**Comment by Smokyish** [comment]
@luisivan Also just realized that there's now bit of repetition with both "`It's designed to do one simple thing exceedingly well`" in first paragraph and "`aragonOS-based applications follow the UNIX philosophy and do one simple thing very well.`" in the second.

**Comment by luisivan** [comment]
@Smokyish true. Feel free to fix that

**Pull request merged**